articles/quickstart/backend/rails/interactive.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Add authorization to a Ruby on Rails API
+description: This tutorial demonstrates how to add authorization to a Ruby on Rails API.
+topics:
+    - quickstart
+    - backend
+    - rails
+github:
+  path: 01-Authentication-RS256
+contentType: tutorial
+useCase: quickstart
+interactive: true
+files:
+  - files/json_web_token
+  - files/secured
+  - files/public
+  - files/private
+---
+<!-- markdownlint-disable MD041 MD025 -->
+
+# Add authorization to a Ruby on Rails API
+This tutorial performs Access Token validation using the  [jwt](https://github.com/jwt/ruby-jwt) Gem. A Concern called `Secured` is used to mark endpoints which require authentication through an incoming Access Token.
```

Comment:
Style guide (word list): access token should be lowercase

```suggestion
This tutorial performs access token validation using the  [jwt](https://github.com/jwt/ruby-jwt) Gem. A Concern called `Secured` is used to mark endpoints which require authentication through an incoming access token.
```

Reply:
TIL